app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ApplicationRecord
   has_secure_password
   has_person_name
 
-  validates :password, length: { minimum: 6 }, allow_nil: true
+  validates :password, length: { minimum: 6 }, allow_nil: Feature.authenticate_with_password?
```

Comment:
It feels backwards that this as ever true — wouldn’t we want to NOT allow nil passwords, if we’re supporting password-based auth?

Reply:
Agreed. My brain was twisted.
`allow_nil: true` should not be here